.travis.yml

```diff
@@ -7,38 +7,61 @@ cache:
 matrix:
     fast_finish: true
     include:
-        - python: "3.6"
+        - name: "Python 3.6 Trusty"
```

Comment:
No, please do not change this.

Travis has separate columns for:
- os (showing penguin or apple)
- dist / os release (showing e.g. trusty or 10.12)
- python (showing the python version, e.g. only "3.6")